climada/hazard/centroids/centr.py

```diff
@@ -22,20 +22,19 @@
 import ast
 import copy
 import logging
-import numpy as np
-from scipy import sparse
+
+import geopandas as gpd
 import h5py
+import numpy as np
 import pandas as pd
-from rasterio import Affine
+import rasterio
 from rasterio.warp import Resampling
-import geopandas as gpd
+import scipy.sparse as sp
```

Comment:
Probably better to keep is as `from scipy import sparse` as this seems to me the prevalent way in the current code base.

Reply:
Agreed and changed!